spec/models/measure_condition_spec.rb

```diff
@@ -154,4 +154,38 @@
       end
     end
   end
+
+  describe '#document_code' do
```

Comment:
@matthewford I know that it's not belong, should I remove spec?
just covering uncovered methods